fhir-bulkimportexport-webapp/src/main/java/META-INF/batch-jobs/FhirBulkImportChunkJob.xml

```diff
@@ -14,6 +14,8 @@
         <chunk checkpoint-policy="item" item-count="#{jobProperties['fhir.cosreadsperdbbatch']}">
             <reader ref="com.ibm.fhir.jbatch.bulkdata.load.ChunkReader">
                 <properties>
+                    <property name="fhir.tenant" value="#{jobParameters['fhir.tenant']}"/>
+                    <property name="fhir.datastoreid" value="#{jobParameters['fhir.datastoreid']}"/>
```

Comment:
how were we passing these before?

Reply:
this is for supporting the new per tenant config, not needed before.